docs/monthly/2023-08-monthly.md

```diff
@@ -159,33 +146,29 @@ noticed, in particular, the following proposals:
 ## Conclusion
 
 The monthly review meeting for August 2023 was held on 2023-08-23 via Google
-Meet with these [slides][slides] and here is the [recording][recording] .. this
-time "broadcasting live from warm & sunny Colorado"!
+Meet with these [slides][slides] and the [recording][recording], 'broadcasting live from warm and sunny Colorado'!
```

Comment:
Why single quotes and not double quotes? (any style guide would be helpful)

Reply:
I don't know why :) But yes, this is according to the IOG style guide:
- Use single quote marks: ‘cascading disruption’, said the duchess to the bishop